tests/run.sh

```diff
@@ -104,34 +100,46 @@ function get_requirements() {
     inject_pre_generated_ssh_keys
 }
 
+# Old ways of getting the image, now deprecated, but still needed for images
+# built with thud or older.
+get_ext4_image_deprecated() {
+    if [[ -n "$BUILDDIR" ]]; then
+        cp -f "$BUILDDIR/tmp/deploy/images/$MACHINE_NAME/core-image-full-cmdline-$MACHINE_NAME.ext4" .
+    elif [[ -n "$DOWNLOAD_REQUIREMENTS" ]]; then
+        curl --fail "https://mender.s3-accelerate.amazonaws.com/temp_${MENDER_BRANCH}/core-image-full-cmdline-$MACHINE_NAME.ext4" \
+             -o core-image-full-cmdline-$MACHINE_NAME.ext4 \
+             -z core-image-full-cmdline-$MACHINE_NAME.ext4
+    fi
+}
+
 if [[ $1 == "--get-requirements" ]]; then
     get_requirements
     exit 0
 fi
 
 dd if=/dev/zero of=large_image.dat bs=300M count=0 seek=1
 
-if [[ -n "$BUILDDIR" ]]; then
-    # Get the necessary path directly from the build.
+# mender-stress-test-client is here
```

Comment:
Just for the record, mender-artifact and the Artifact generators are also here. These are all the dependencies for the integ tests, really.